Formula/llvm.rb

```diff
@@ -272,6 +272,13 @@ def install
     # install llvm python bindings
     (lib/"python2.7/site-packages").install buildpath/"bindings/python/llvm"
     (lib/"python2.7/site-packages").install buildpath/"tools/clang/bindings/python/clang"
+
+    # Install symlinks for omp.h/libomp so they can be used in other formulae
```

Comment:
`# Install symlinks for OpenMP header and libraries so they can be used by superenv`